### DISCUSSION TITLE
Is there a way to give background image to QRStamps?

### DISCUSSION THREAD

**Discussion by Vishwamithra37** [MatthiasValvekens/pyHanko#116] [Q&A] [answered]
Hey, I am trying to give background image with QRStamps with border width zero. But I can not find the option and it is not working. Can anyone show an example?

**Comment by MatthiasValvekens** [comment, marked as answer]
So you're saying that using the `background=` option [as in this example](https://pyhanko.readthedocs.io/en/latest/lib-guide/signing.html#text-based-stamps) doesn't work for QR stamps?
